.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -6,11 +6,11 @@ type: 'Bug'
 labels: ['to triage']
 
 ---
-### Describe the bug
+## Describe the bug
```

Comment:
Alternatively, we could disable the newly added frontmatter support:
https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md

Either globally or only for this file (but this requires an HTML comment in the file and I don't know if it would appear when creating a new issue).